---
title: "Resume"
---

***

### SUMMARY

Ph.D.-trained biomedical data scientist with expertise in bioinformatics, data pipeline development, and high-dimensional analytics. Experienced in next-generation sequencing analysis (bulk, single-cell, spatial), multi-omic data integration, scalable workflow automation, statistical and AI-driven modeling, and data visualization. Effective collaborator and communicator with a track record of delivering results and developing solutions across teams and stakeholders.

### EDUCATION

- **Ph.D., Integrative Biology:** University of Wisconsin-Madison (2018)
- **B.S., Biopsychology (*cum laude*):** Tufts University (2013)

### TECHNICAL SKILLS

- **Programming:** R, Python, Bash, SQL, Julia, Git, Markdown, LaTeX
- **NGS Analysis:** bulk RNA-seq, single-cell RNA-seq, spatial transcriptomics, clustering (graph-based, partition-based, hierarchical), dimensionality reduction (PCA, UMAP, t-SNE), differential expression analysis, linear modeling, 10x Genomics (Chromium, Visium, Cell Ranger, Space Ranger), Seurat
- **Libraries:** tidyverse, ggplot2, shiny, quarto, pandas, Numpy, SciPy, scikit-learn, Plotly, Dash
- **HPC/Cloud Computing:** Linux, GCP, AWS, HDFS

### EXPERIENCE

**Associate Consultant** (Jul. 2025 – Present)\
**ZS Associates**  (Boston, MA)

- Design and deliver solutions for clients across the life sciences including bioinformatics, pharmaceuticals, biotechnology, and biomedical research

**Senior Bioinformatics Scientist** (Jan. 2022 – Oct. 2024)\
**Parexel International**  (Remote)

- Streamlined latency of data delivery by 10x for a major pharmaceutical client by developing novel data ingestion pipelines in R, Python, and Bash to drive biomarker discovery
- Collaborated across bioinformatics and data engineering teams to deliver 1000s of genomic datasets (GWAS & eQTL) scaling to terabytes in size, integrating data and metadata from UK Biobank, 1000 Genomes Project, FinnGen, Open Targets, GWAS Catalog, and more
- Automated high-throughput quality control workflows to enable 24-hour client turnaround

**Postdoctoral Researcher** (Jun. 2018 – Jan. 2022) \
**University of Texas Southwestern Medical Center**  (Dallas, TX)
  
- Led a multi-group collaboration to publish novel single-cell RNA-seq datasets in neurobiology, resulting in a co-first-author paper in *Science* (145+ citations) and a second-author paper in *Nature Communications* (30+ citations)
- Independently analyzed large-scale single-cell RNA-seq data by applying computational pipelines, dataset integration, unsupervised clustering (k-nearest neighbor + Louvain method), and statistical modeling
- Delivered spatial transcriptomics capabilities to research group, encompassing molecular library preparation, data and analytics pipelines, documentation, and training for new users

### INDEPENDENT PROJECTS

**Project ESPAÑOL** [(https://github.com/dmerullo/project-espanol)](https://github.com/dmerullo/project-espanol){target="_blank"}

- Generated a dataset of 10,000+ Spanish-language poems (>2.5 million words) using high-throughput web scraping (rvest)
- Developed an unsupervised clustering (k-means) pipeline in Python to classify poems into 4 difficultly levels based on verb usage
- Built an interactive Plotly Dash web application for text exploration and visualization
  
### SELECTED PUBLICATIONS (700+ citations total)

- Colquitt, B. M.\*, <strong>Merullo, D. P.\*</strong>, Konopka, G., Roberts, T. F., & Brainard, M. S. (2021). Cellular transcriptomics reveals evolutionary identities of songbird vocal circuits. *Science*, 371(6530), eabd9704. <strong>*co-first author</strong>

- Xiao, L., **Merullo, D. P.**, Koch, T. M. I., Cao, M., Co, M., Kulkarni, A., Konopka, G., & Roberts, T. F. (2021). Expression of FoxP2 in the basal ganglia regulates vocal motor sequences in the adult songbird. *Nature Communications*, 12(1), 2617.

- Kulkarni, A., Anderson, A. G., **Merullo, D. P.**, & Konopka, G. (2019). Beyond bulk: A review of single cell transcriptomics methodologies and applications. *Current Opinion in Biotechnology*, 58, 129–136.

- **Merullo, D. P.**, Asogwa, C. N., Sanchez-Valpuesta, M., Hayase, S., Pattnaik, B. R., Wada, K., & Riters, L. V. (2018). Neurotensin and neurotensin receptor 1 mRNA expression in song-control regions changes during development in male zebra finches. *Developmental Neurobiology*, 78(7), 671–686.

- **Merullo, D. P.**, Spool, J. A., Zhao, C., & Riters, L. V. (2018). Co-localization patterns of neurotensin receptor 1 and tyrosine hydroxylase in brain regions involved in motivation and social behavior in male European starlings. *Journal of Chemical Neuroanatomy*, 89, 1–10.

- **Merullo, D. P.**, Angyal, C. S., Stevenson, S. A., & Riters, L. V. (2016). Song in an affiliative context relates to the neural expression of dopamine- and neurotensin-related genes in male European starlings. *Brain, Behavior and Evolution*, 88, 81–92.

- **Merullo, D. P.**, Cordes, M. A., DeVries, M. S., Stevenson, S. A., & Riters, L. V. (2015). Neurotensin neural mRNA expression correlates with vocal communication and other highly-motivated social behaviors in male European starlings. *Physiology and Behavior*, 151, 155–161.

- **Merullo, D. P.**, Cordes, M. A., Stevenson, S. A., & Riters, L. V. (2015). Neurotensin immunolabeling relates to sexually-motivated song and other social behaviors in male European starlings (Sturnus vulgaris). *Behavioural Brain Research*, 282, 133–143.